tf-modules/sqs-message-remover-lambda/tests/test-index.js

```diff
@@ -8,11 +8,85 @@ const get = require('lodash/get');
 const awsServices = require('@cumulus/aws-client/services');
 const { receiveSQSMessages } = require('@cumulus/aws-client/SQS');
 const { randomString } = require('@cumulus/common/test-utils');
-const { createSqsQueues, getSqsQueueMessageCounts } = require('../../lib/testUtils');
 
-const sqsMessageRemover = rewire('../../lambdas/sqs-message-remover');
+const sqsMessageRemover = rewire('..');
 const updateSqsQueue = sqsMessageRemover.__get__('updateSqsQueue');
 
+// TODO: Copied from @cumulus/api/lib/testUtils to avoid the dependency, but
```

Comment:
This exposes one difficulty with this approach of a "module per Lambda" approach - we have lots of these generic helpers for testing or that are actually used by the Lambda code and which are shared across multiple Lambdas. So where should those files live?

Reply:
Probably a separate internal package.